packages/flutter_tools/lib/src/commands/apk.dart

```diff
@@ -19,6 +19,7 @@ import 'start.dart';
 
 const String _kDefaultAndroidManifestPath = 'apk/AndroidManifest.xml';
 const String _kDefaultOutputPath = 'build/app.apk';
+const Strong _kDefaultResourcesPath = 'apk/res';
```

Comment:
you have a typo here, Strong instead of String. I think that's what travis is mad about.